libs/client/src/lib/client.ts

```diff
@@ -31,7 +31,7 @@ export class Client {
     dataCallback: (task, data, cached) => {},
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     errorCallback: (task, data) => {},
-    pollingDelayMs: 1000,
+    pollingDelayMs: 5000,
```

Comment:
why you changed this ?

Reply:
I increased value for pullingDelayMS from 1 second to 5. This is to reduce load on tskmgr server when all available tasks are in work,    others are blocked and there is available runners waiting for a task. For example I have 8 runners 6 parallel 48 total. Is one occupied with one task which is dependency for all other tasks, 47 instances will hit server for a task every 1 second, which could make server untouchable at some point.

Reply:
this is basically why I'm saying the added code is not optimal. figuring out which task among 200 tasks should be instant and not causing such performance issue we currently have.